src/modules/home/profile/elements/skills/profile-skills_module.tsx

```diff
@@ -41,8 +42,14 @@ interface ProfileSkillsDispatch {
 
 class ProfileSkillsModule extends React.Component<ProfileSkillsProps & ThemeProps & ProfileSkillsLocalProps & ProfileSkillsDispatch, ProfileSkillsLocalState> {
 
+
+    // TODO selected skill nach änderung neu setzten um alte referenz zu löschen
```

Comment:
Das scheint nicht mehr notwendig zu sein, oder? Falls doch: Lass das doch in den Store werfen. Das löst das Problem ziehmlich einfach.